### DISCUSSION TITLE
Is there a new placeholder for newsletter confirmation?

### DISCUSSION THREAD

**Discussion by refugeindhamma** [liedekef/events-made-easy#44] [Q&A] [answered]
The placeholder I placed in the confirmation email for newsletter is `#_SUB_CONFIRM_URL`. But it says "This link is not (or no longer) valid" for one email. 
![signal-2023-01-29-095052_002](https://user-images.githubusercontent.com/101639308/215299969-58012746-816b-44d6-a936-5fd999678cc2.png)

For another email, it says "Subscription failed".
![Screenshot 2023-01-29 at 09-48-43 Untitled](https://user-images.githubusercontent.com/101639308/215299888-6caac462-474b-42a1-b2ba-03065c619501.png)

My settings:
![Newsletter subscription](https://user-images.githubusercontent.com/101639308/215300031-79cbf537-95c1-480a-bce6-64d2323cd729.png)

Is this a bug? Should I raise an issue for this? Or is there a new placeholder for subscription confirmation?

**Comment by liedekef** [comment]
It might be that I'm checking something too strict, let me check.

**Comment by liedekef** [comment]
I'm unable to reproduce this as an issue ... can you recreate this on a testsite and give me access?

**Comment by refugeindhamma** [comment]
I have already sent the link to your email. I also encountered another issue when setting up the test server for your access. Please take a look.

**Comment by liedekef** [comment, marked as answer]
Confirmed as a bug: when a person not in EME tried to register for a mailing, it failed (due to too much restrictions on a check later on). Fixed in 2.3.46

**Comment by refugeindhamma** [comment]
Thanks for the fix!
